web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/Plugin/views/filter/ViewsBasicFilter.php

```diff
@@ -24,10 +24,15 @@ public function query() {
     $query = $this->query;
 
     // Parse content type filters.
-    // @todo Add taxonomy term as a filter. Code here: https://gist.github.com/marcb4k/fd4716eb133aa846f90a9e42c80def8c.
     foreach ($this->value['filters']['types'] as $content_type) {
       $query->addWhere($this->options['group'], 'type', $content_type);
     }
+
+    // Parse tag filters.
+    $tag_table = $query->addTable('node__field_tags');
```

Comment:
WHAT? How did I not know `addTable` exists? I've been writing joins for years like a chump.

Reply:
It had been a while since I've done queries like that, so I looked it up and saw that - pretty neat! I recall doing joins before as well.